### PR TITLE
standalone: Replace results JSON textarea with copy button

### PR DESCRIPTION
**Pull request by kainino0x**
In Chromium, this textarea causes a massive relayout whenever it gets
updated (e.g. 40s on my machine). Remove it, and replace it with a button
that copies the JSON to the clipboard (fast).

### CODE REVIEW

**Comment by kainino0x** [comment]
@egalli PTAL!

File: standalone/index.html [hunk 1]
```diff
@@ -216,7 +216,10 @@ <h1>WebGPU Conformance Test Suite</h1>
 
     <div id="info"></div>
     <div id="resultsVis"></div>
-    <textarea id="resultsJSON" spellcheck="false"></textarea>
+
+    <p>
+      <input type=button id=copyResultsJSON value="Copy results as JSON">
```
Comment (austinEng) [inline standalone/index.html:221]: nit: quotes on the html attributes?

**Review by austinEng — APPROVED** [review]

File: standalone/index.html (hunk 1 above)
Comment (kainino0x) [reply standalone/index.html:221]: Elsewhere in this project I have been not using quotes (src/common/templates/cts.html) but sg, let's just keep this one internally consistent for now.

**Review by egalli — APPROVED** [review]
LGTM

**Pull request merged**